zerver/migrations/0375_set_realmemoji_author.py

```diff
@@ -0,0 +1,34 @@
+from django.db import migrations
+from django.db.backends.postgresql.schema import DatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+
+def set_emoji_author(apps: StateApps, schema_editor: DatabaseSchemaEditor) -> None:
+    """
+    This migration establishes the invariant that all RealmEmoji objects have .author set.
+    """
+    RealmEmoji = apps.get_model("zerver", "RealmEmoji")
+    UserProfile = apps.get_model("zerver", "UserProfile")
+    ROLE_REALM_OWNER = 100
+
+    realm_emoji_to_update = []
+    for realm_emoji in RealmEmoji.objects.all():
+        if realm_emoji.author_id is None:
+            user_profile = UserProfile.objects.filter(
+                realm_id=realm_emoji.realm_id, is_active=True, role=ROLE_REALM_OWNER
+            ).first()
```

Comment:
Let's include an `order_by("id")` here -- we want to be specific about what "first" means.

Reply:
Yeah, it's more a style thing -- if you care about the ordering, you should write it.